dataline-integrations/singer/postgres/destination/docker/config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
what is this file?

Reply:
Can you generate it in the test? Potentially using the testcontainer info? Let's keep these directories clean.

Reply:
oh woops not intended to be checked in. removed